resources/profiles/Geeetech/filament/Generic ASA @Geeetech.json

```diff
@@ -0,0 +1,17 @@
+{
+    "type": "filament",
+    "filament_id": "GFB98",
```

Comment:
Bambu Studio has strict filament_id rule, check the filament_id of BBL, the same filament uses the same filament_id. For example, the filament_id of ABS is GFSB00

Reply:
Revised filament_id as requested